apps/agents/test/test_text_utils.py

```diff
@@ -0,0 +1,112 @@
+from unittest import TestCase
+
+import apps.agents.text_utils as text_utils
+
+
+class TestTextUtils(TestCase):
+    def test_split_markdown_code_newline(self):
+        self.assertTrue(True)
```

Comment:
Could you explain what this line is doing?

Reply:
Removed the leftover